.appveyor.yml

```diff
@@ -40,7 +43,8 @@ test_script:
       "--------------------------------------------"
       "Running npm run test:without-system-tests..."
```

Comment:
Can you update this message to reflect which matrix option is running ?

Reply:
Yup, will do.

Reply:
Actually, I haven't been able to configure it to display different messages based on which matrix option is running. For now, I can update the message to display both the options?

Reply:
Did you try changing `"Running npm run ... "` to `echo Running npm run %vscode%` ? If that doesn't work then we can look at it later.

Reply:
That worked!